Brewfile

```diff
@@ -1,4 +1,8 @@
 # Brewfile
+tap "tilt-dev/tap"
+
 brew "kubectl" # For kubectl
 brew "kustomize" # For kustomize
-brew "helm" # For helm
\ No newline at end of file
+brew "helm" # For helm
+brew "tilt-dev/tap/tilt" # For Tilt
```

Comment:
```suggestion
brew "tilt"
```

If you're defining `tap` above like on line 2, then you don't have to define the full path here IIRC.